### PR TITLE
chore: remove usage of chrono::Utc::now()

### PR DESCRIPTION
**Pull request by littledivy**
Remove usage of Chrono's clock feature which pulls in iana-time-zone -> core-foundation

### CODE REVIEW

File: cli/tools/jupyter/jupyter_msg.rs [hunk 1]
```diff
@@ -15,6 +14,8 @@ use ring::hmac;
 use std::fmt;
 use uuid::Uuid;
 
+include!("../../util/time.rs");
```
Comment (dsherret) [inline cli/tools/jupyter/jupyter_msg.rs:17]: I think this will fail to cargo publish?

Comment (dsherret) [reply cli/tools/jupyter/jupyter_msg.rs:17]: I mean, below in ext/kv it will fail

Comment (bartlomieju) [reply cli/tools/jupyter/jupyter_msg.rs:17]: Agreed, maybe just duplicate it for now?

Comment (littledivy) [reply cli/tools/jupyter/jupyter_msg.rs:17]: Added duplicate in ext/kv

Comment (dsherret) [inline cli/tools/jupyter/jupyter_msg.rs:17]: Just noticed this, but this one could be imported instead of included.

File: Cargo.toml [hunk 2]
```diff
@@ -78,7 +78,7 @@ brotli = "3.3.4"
 bytes = "1.4.0"
 cache_control = "=0.2.0"
 cbc = { version = "=0.1.2", features = ["alloc"] }
-chrono = { version = "0.4", default-features = false, features = ["std", "serde", "clock"] }
+chrono = { version = "0.4", default-features = false, features = ["std", "serde"] }
```
Comment (dsherret) [inline Cargo.toml:81]: Maybe add a comment that people should not add the "clock" feature and outline the reasons why. That will help prevent it being added again.

**Review by dsherret — APPROVED** [review]
LGTM!

**Pull request merged**